File: java/triangle.java
import java.util.Scanner;

public class triangle {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int count = 1;
		while(scan.hasNextInt()) {
			int n = scan.nextInt();
			int digits = (int) Math.ceil(n * Math.log10(1.5) + Math.log10(3));
			System.out.println("Case " + count++ + ": " + digits);
		}
		scan.close();
	}
}

File: ts/src/triangle.ts
import readline = require('readline')

function getTriangleCount (num: number) {
  return Math.ceil(num * Math.log10(1.5) + Math.log10(3))
}

let lineCount = 0

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    console.log(`Case ${++lineCount}: ${getTriangleCount(parseInt(line))}`)
  })
  .on('close', () => {
    process.exit(0)
  })
